perllib/FixMyStreet/Cobrand/Enfield.pm

```diff
@@ -0,0 +1,91 @@
+package FixMyStreet::Cobrand::Enfield;
+use parent 'FixMyStreet::Cobrand::UKCouncils';
+
+sub council_area_id { 2495 }
+sub council_area { 'Enfield'; }
+sub council_name { return 'Enfield Council'; }
+sub council_url { return 'enfield'; }
+sub base_url { return FixMyStreet->config('BASE_URL'); }
+
+sub open311_config {
+    my ($self, $row, $h, $params, $contact) = @_;
+
+    $params->{multi_photos} = 1;
+    $params->{upload_files} = 1;
+}
+
+sub open311_update_missing_data {
+    my ($self, $row, $h, $contact) = @_;
+
+    if ($row->get_extra_field_value('pac')) {
+        if (my $result = $self->_lookup_os('uprn', $row)) {
+            if (my $uprn = $result->{LPI}{UPRN}) {
+                $row->update_extra_field({ name => 'uprn', description => 'UPRN', value => $uprn });
+            }
+        }
+    } else {
+        if (my $result = $self->_lookup_os('usrn', $row)) {
+            if (my $usrn = $result->{LPI}{USRN}) {
+                $row->update_extra_field({ name => 'usrn', description => 'USRN', value => $usrn });
+            }
+        }
+    }
```

Comment:
Both branches of this if statement are doing basically the same thing. Think it could be expressed more concisely as:

```suggestion
    my $field = $row->get_extra_field_value('pac') ? 'uprn' : 'usrn';

    if (my $result = $self->_lookup_os($field, $row)) {
        if (my $value = $result->{LPI}{uc $field}) {
            $row->update_extra_field({ name => 'usrn', description => uc $field, value => $usrn });
        }
    }
```